pkg/consts/consts.go

```diff
@@ -129,14 +134,14 @@ const (
 	// QueryParamUpdatedAt is the key for the `Updated_at` value in a query
 	// string.
 	QueryParamUpdatedAt = "Updated_at"
-	// QueryParamReferencedBy is the key for the `referenced_by` values in a
-	// query string.
-	QueryParamReferencedBy = "Referenced_by"
 	// QueryParamRecursive is the key for the `recursive` value in a query
 	// string.
 	QueryParamRecursive = "Recursive"
 	// QueryParamTags is the key for the `tags` values in a query string.
 	QueryParamTags = "Tags"
+	// QueryParamReferencedBy is the key for the `referenced_by` values in a
+	// query string.
+	QueryParamReferencedBy = SelectorReferencedBy
```

Comment:
In JSON-API, only some reserved query parameters are allowed to be in lowercase. For most of them, it should start with an upper case.